brave/src/test/java/brave/sampler/RateLimitingSamplerTest.java

```diff
@@ -92,17 +88,6 @@
     assertThat(sampler.isSampled(0L)).isTrue();
   }
 
-  @DataPoints public static final int[] SAMPLE_RESERVOIRS = {1, 10, 100};
-
-  @Theory public void retainsPerSampleRate(int reservoir) {
```

Comment:
this didn't run as it needed a test runner. I guess we added the powermock runner later and didn't notice